packages/create-nx-workspace/src/utils/package-manager.ts

```diff
@@ -2,6 +2,8 @@ import { execSync } from 'child_process';
 import { existsSync, writeFileSync } from 'fs';
 import { join } from 'path';
 
+import { sep } from 'node:path';
```

Comment:
Let's not have separate imports to `path`. Please combine this with the import on line 3 (using `node:path` for both `join` and `sep`)

Reply:
Ah I see you most recently worked on this @xiongemi maybe you could address my points above please

Reply:
Yes - it's a big thanks to @xiongemi . I was not able to figure out why tests were failing